### PR TITLE
Updated Xenial LTS version to 16.04.7, as 16.04.6 deprecated.

### PR DESCRIPTION
**Pull request by zenmonkeykstop**
## Status

Ready for review 

## Description of Changes

* Description: Updates references to Ubuntu LTS version to 16.04.7, as 16.04.6 is no longer available.

## Testing
- Are relevant sections of docs updated? (server setup, qubes staging env setup)

### CODE REVIEW

File: docs/servers.rst [hunk 1]
```diff
@@ -108,13 +108,13 @@ key") means that you are not ready to proceed. ::
 
 The next and final step is to verify the Ubuntu image. ::
 
-    sha256sum -c <(grep ubuntu-16.04.6-server-amd64.iso SHA256SUMS)
+    sha256sum -c <(grep ubuntu-16.07.6-server-amd64.iso SHA256SUMS)
```
Comment (eloquence) [inline docs/servers.rst:111]: wrong number updated here :)

File: docs/servers.rst [hunk 2]
```diff
@@ -108,13 +108,13 @@ key") means that you are not ready to proceed. ::
 
 The next and final step is to verify the Ubuntu image. ::
 
-    sha256sum -c <(grep ubuntu-16.04.6-server-amd64.iso SHA256SUMS)
+    sha256sum -c <(grep ubuntu-16.06.7-server-amd64.iso SHA256SUMS)
```
Comment (eloquence) [inline docs/servers.rst:111]: should be `16.04.7` not `16.06.7`

Comment (zenmonkeykstop) [reply docs/servers.rst]: I obviously can't computer today.

**Comment by eloquence** [comment]
In this branch, the intro for `servers.rst` still says:
> Please ensure you are using Ubuntu Xenial ISO images 16.04.6 or greater. Ubuntu Xenial ISO images 16.04.5 and lower ship with a version of the apt package vulnerable to CVE-2019-3462. If you are using 16.04.5 or lower, the initial base OS must be installed without Internet connectivity.

How about we change this to a more general warning to use the latest ISO to ensure security fixes are in place at the time of the install?

**Review by eloquence — APPROVED** [review]
LGTM :-)

**Pull request merged**